app/src/main/java/com/github/se/icebreakrr/ui/sections/shared/TopBar.kt

```diff
@@ -68,9 +68,10 @@ fun TopBar(s: String, needBackButton: Boolean = false, backButtonOnClick: () ->
                       modifier = Modifier.size(BACK_BUTTON_ICON_SIZE))
                 }
           }
+          val topBarSize = if (s.length < 15) TOP_BAR_TEXT_SIZE else 25.sp
```

Comment:
Why 15? It's a magic number!

Reply:
True! It is because over 15 character, the text uses 2 lines in the top app bar. I've put these into const variables